pkg/projectfile/projectfile.go

```diff
@@ -1185,6 +1185,29 @@ func (p *Project) Persist() {
 	os.Setenv(constants.ProjectEnvVarName, p.Path())
 }
 
+type configGetter interface {
+	GetStringMapStringSlice(key string) map[string][]string
+}
+
+func GetProjectNameForPath(config configGetter, projectPath string) string {
+	projects := config.GetStringMapStringSlice(LocalProjectsConfigKey)
+	if projects == nil {
+		projects = make(map[string][]string)
+	}
+
+	for name, paths := range projects {
+		if name == "/" {
+			continue
+		}
+		for _, path := range paths {
+			if isEqual, _ := fileutils.PathsEqual(projectPath, path); isEqual {
```

Comment:
We should at least be logging the error here, if not returning it.

Reply:
I logged it.